src/common/cta/index.tsx

```diff
@@ -0,0 +1,16 @@
+import S from "./Cta.module.scss";
+
+interface CtaProps {
+  children: React.ReactNode;
+  buttonType?: "button" | "submit";
+}
+
+const Cta = ({ children, buttonType = "button" }: CtaProps) => {
+  return (
+    <button type={buttonType} className={S.container}>
+      {children}
+    </button>
+  );
+};
```

Comment:
기본적인 button element의 attribute 타입을 이런 식으로 가져올 수도 있습니다. 참고해 주세요!

```suggestion
interface CtaProps extends ButtonHTMLAttributes<HTMLButtonElement> {}

const Cta = ({ children, type = "button" }: CtaProps) => {
  return (
    <button type={type} className={S.container}>
      {children}
    </button>
  );
};
```